airbyte-webapp-e2e-tests/cypress/integration/source.spec.ts

```diff
@@ -1,16 +1,26 @@
-import { appendRandomString } from "commands/common";
+import { appendRandomString, submitButtonClick } from "commands/common";
 import { createPostgresSource, deleteSource, updateSource } from "commands/source";
 import { initialSetupCompleted } from "commands/workspaces";
+import { goToSourcePage, openNewSourceForm } from "pages/sourcePage";
+import { openHomepage } from "pages/sidebar";
+import { selectServiceType } from "pages/createConnectorPage";
+import { fillPokeAPIForm } from "commands/connector";
+import { should } from "chai";
+
 
 describe("Source main actions", () => {
   beforeEach(() => {
     initialSetupCompleted();
   });
 
   it("Create new source", () => {
+    cy.intercept("/api/v1/sources/create").as("createSource");
     createPostgresSource("Test source cypress");
 
-    cy.url().should("include", `/source/`);
+    cy.wait("@createSource", {timeout: 30000}).then((interception) => {
+      assert("include", `/source/` + interception.response?.body.Id)});
+
+    //cy.url().should("include", `/source/`);
```

Comment:
Does 
```ts
assert("include", `/source/${interception.response?.body.Id}`)`
```
assert against the url? (As an aside, if you're using backticks, you can insert values into the string with code, like `${value}`, which is nicer to read) It would be a bit more readable to keep the old assertion style:
```ts
cy.wait("@createSource", { timeout: 30000 }).then((interception) => {
  cy.url().should("include", `/source/${interception.response?.body.Id}`);
});
```

Reply:
> assert against the url?

Yeah, that's actually my suggestion during short sync with @SofiiaZaitseva. After connector creation, we aim to check that we were redirected to the "Overview" page.